qa/os/src/test/java/org/elasticsearch/packaging/test/DockerTests.java

```diff
@@ -97,9 +98,7 @@ public void teardownTest() {
     /**
      * Checks that the Docker image can be run, and that it passes various checks.
      */
-    public void test010Install() throws Exception {
-        // Wait for the container to come up, because we assert the state of some files that Elasticsearch creates on startup.
-        waitForElasticsearch(installation);
```

Comment:
Are there no other files that get laid down on startup? It's just the keystore?

Reply:
Besides log files, it's just the keystore AFAIK.